File: 344-Reverse-String.cs
public class Solution {
    public void ReverseString(char[] s) 
    {
        int x = s.Length;
        for(int i = 0 ; i < x / 2 ; i++ )
        {
            char c = s[i] ;
            s[i] = s[ x - i - 1] ;
            s[x - i- 1] = c ;
        }
    }
}

File: 344-Reverse-String.java
class Solution {
    public void reverseString(char[] s) {
        int x = 0 ;
        int y = s.length - 1;
        while(x < y)
        {
            char c = s[y] ;
            s[y] = s[ x ] ;
            s[x] = c ;

            x++ ;
            y-- ;
        }
    }
}